features/include/pcl/features/impl/linear_least_squares_normal.hpp

```diff
@@ -45,8 +45,7 @@
 //////////////////////////////////////////////////////////////////////////////////////////////
 template <typename PointInT, typename PointOutT>
 pcl::LinearLeastSquaresNormalEstimation<PointInT, PointOutT>::~LinearLeastSquaresNormalEstimation ()
-{
-}
+= default;
```

Comment:
You might want to review your own PR and remomve all these default destructors. I

Reply:
There were a bunch in the code that were made visible by being changed to `= default`.

What criteria are you using to decide when you want the compiler to create one implicitly?

Reply:
I suppose all of the ones tagged `Empty destructor` would fit...

Reply:
With destructors I use the following policy:
- if your class defines virtual methods which it defines (literally defining, not overriding), I explicitly default it. The compiler throws warnings telling if of your class has virtual methods but the destructor is not virtual.
- If a class has empty / default destructors I just remove those lines of code. If the destructor is not implicitly generated you'll get a compiler error, so there's no chance you'll mess up.


For constructors:
- I remove all default constructors unless there's a user specified (non-default) constructor.

Reply:
Thanks, sounds like a good focus for another, separate, PR.

Reply:
Split off as a separate task for #4283